test/lib/fixtures/out-of-sync-tree/expected-tree.json

```diff
@@ -0,0 +1,192 @@
+{
+  "dependencies": {
+    "debug": {
+      "depType": "prod",
+      "dependencies": {
+        "ms": {
+          "depType": "prod",
+          "dependencies": {},
+          "name": "ms",
+          "version": "2.1.1"
+        }
+      },
+      "name": "debug",
+      "version": "4.1.0"
+    },
+    "body-parser": {
+      "depType": "prod",
+      "missingLockFileEntry": true,
```

Comment:
all dependencies will need to be filtered out on the Registry side, we can also there have the logic of using the filtered out deps as a user message to show the user